docs/src/modules/components/AdCarbon.js

```diff
@@ -123,7 +123,7 @@ export function AdCarbonInline(props) {
       />
     </React.Fragment>
   ) : (
-    <div {...props} style={{ minHeight: 45 }} />
+    <div {...props} style={{ minHeight: 52 }} />
```

Comment:
To match 

<img width="620" alt="Screenshot 2022-11-12 at 17 25 59" src="https://user-images.githubusercontent.com/3165635/201484084-abb8c3fc-a941-4aac-b43f-3d6449e9dc65.png">